tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -774,6 +774,29 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 		framework.ExpectEqual(existed, true)
 		framework.ExpectEqual(value, label.TiDBOperator)
 	})
+
+	ginkgo.It("tidb-scale: clear TiDB failureMembers when scale TiDB to zero", func() {
+		cluster := newTidbClusterConfig(e2econfig.TestConfig, ns, "tidb-scale", "admin", "")
+		cluster.Resources["pd.replicas"] = "3"
+		cluster.Resources["tikv.replicas"] = "1"
+		cluster.Resources["tidb.replicas"] = "1"
+
+		oa.DeployTidbClusterOrDie(&cluster)
+		oa.CheckTidbClusterStatusOrDie(&cluster)
+		oa.CheckDisasterToleranceOrDie(&cluster)
+		// generate failmembers
+		cluster.TiDBPreStartScript = strconv.Quote("exit 1")
+		_, cancel := context.WithCancel(context.Background())
+		oa.UpgradeTidbClusterOrDie(&cluster)
+		oa.CheckTidbClusterHaveFailedMemberOrDie(&cluster)
+		//scale tidb member to zero replica
```

Comment:
instead of commenting, use `ginkgo.By(` to log the process of e2e test, e.g. `ginkgo.By("scaling the replicas of tidb member to zero")`